ext/transaction-process/templates/money-paid/money-paid-html.html

```diff
@@ -69,8 +69,8 @@ <h1>You have been paid {{> format-money money=payout-total}}</h1>
           <th class="right">{{> format-day-time date=booking.end}}</th>
         </tr>
         <tr class="bottom-row">
-          <th class="left">{{> format-month-date date=booking.start format="MMM d"}}</th>
-          <th class="right">{{> format-month-date date=booking.end format="MMM d"}}</th>
+          <th class="left">{{> format-month-date date=booking.start}}</th>
```

Comment:
What does this removal of extra "format" parameter does (or what was the effect previously)?

Reply:
Ah, it actually was extra (I didn't open collapsed lines before commenting)

Reply:
Yup, it didn't have any effect.